src/code/views/inspector-tabs-view.coffee

```diff
@@ -0,0 +1,14 @@
+{div} = React.DOM
+module.exports = React.createClass
+
+  displayName: 'InspectorTabsView'
+
+  render: ->
+    (div {className: 'inspector-tabs'},
+      for tab in @props.tabs
+        if tab is @props.selected
+          (div {className: 'inspector-tab selected'},tab)
+        else
+          (div {className: 'inspector-tab'},tab)
+
```

Comment:
Does this generate a key warning in React since you are building this in a for loop?